src/Utility/ElementBlock.C

```diff
@@ -20,11 +20,12 @@
 
 double ElementBlock::eps = 0.0;
 
+static std::array<size_t,7> legalNENs = { 2, 3, 4, 6, 8, 9, 37 };
```

Comment:
37 vs 27 in original code?

Reply:
bummer, already fixed. Should be 27 of course (9^3)